rpm/sailfish-setup.spec

```diff
@@ -12,11 +12,23 @@ Requires(pre): setup
 # Some mandatory groups are created by systemd like input
 Requires: systemd
 Requires(pre): systemd
+# Commands used by scriptlets
 Requires(pre): /usr/bin/getent
 Requires(pre): /usr/sbin/groupadd
 Requires(pre): /usr/sbin/useradd
 Requires(pre): /usr/sbin/usermod
+# Needed by manage-groups.sh
+Requires: /usr/bin/getent
+Requires: /usr/sbin/groupadd
+Requires: /usr/sbin/groupdel
+Requires: /usr/sbin/usermod
+Requires: coreutils
+Requires: grep
+# Other
 Recommends: hardware-adaptation-setup
+# group.ids format changed
+Conflicts: user-managerd < 0.8.1
```

Comment:
Hmm. Actually might be better to leave these out, I'm not 100% sure they'll work as I intended.

Reply:
I got a better idea. I'll just rename that group.ids to group_ids.env. This means also that old code doesn't even try to use it and now it has name that better describes the file format. I think then it's fine without the conflicts too. The new packages should get installed anyway.